### PR TITLE
Added JSS styling for CaregroupCard Component

### PR DESCRIPTION
**Pull request by evantaurusta**
screen shot 2018-07-21 at 3 59 03 pm

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
 #main {
   margin: 0 10%;
 }
+
+.body {
+  text-align: center;
```
Comment (sdkitono) [inline src/App.css]: why need text-align center in body?

File: src/components/Card/CaregroupCard.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      });
+    });
+  }
+  /**
+   * Populate card with data from the api
+   */
+  populate = () => {
+    const {classes} = this.props;
+    if (this.state.caregroups.length === 0) {
+      return null;
```
Comment (sdkitono) [inline src/components/Card/CaregroupCard.js]: if you already using map do we still need to explicitly return null

File: src/containers/Caregroup/Caregroup.js [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
-                <CareGroupCard/>
-            </div>
-        );
-    }
+  render() {
+    return (
+      <React.Fragment>
+        <h1>Care Group</h1>
+        <div className='caregroup'>
+          <CaregroupCard />
```
Comment (sdkitono) [inline src/containers/Caregroup/Caregroup.js]: Hmm this is weird why you dont put the map of the states in here instead? The right approach is to make CareGroupCard a function component instead that receives props and just render it without business logic. You should put the business logic in this component instead

**Review by sdkitono — CHANGES_REQUESTED** [review]
Please see comments above for the changes. You can ask me for further details.

File: src/containers/Caregroup/Caregroup.js (hunk 3 above)
Comment (evantaurusta) [reply src/containers/Caregroup/Caregroup.js]: like this? https://www.youtube.com/watch?v=Lx0AE-mhNlg

File: src/components/Card/CaregroupCard.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        <CardContent className={classes.content}>
+          <Typography className={classes.title} variant='title'>
+            {caregroup.Name}
+          </Typography>
+          <Typography variant='subheading'>
+            {caregroup.ShortName}
+            {caregroup.Description}
+          </Typography>
+        </CardContent>
+
```
Comment (sdkitono) [inline src/components/Card/CaregroupCard.js]: Remove spacing

**Review by sdkitono — APPROVED** [review]

**Pull request merged**